### PR TITLE
Change references from Rake task to Rails command

### PR DESCRIPTION
**Pull request by albertoalmagro**
### Summary
This commit follows the path we started at commit ea4f0e2 and continued at PR #33229.

Thanks for your review!

### CODE REVIEW

File: railties/lib/rails/source_annotation_extractor.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@ def to_s(options = {})
       # Used in annotations.rake
       #:nodoc:
       def self.notes_task_deprecation_warning
-        ActiveSupport::Deprecation.warn("This rake task is deprecated and will be removed in Rails 6.1. \nRefer to `rails notes --help` for more information.\n")
+        ActiveSupport::Deprecation.warn("This Rails command is deprecated and will be removed in Rails 6.1. \nRefer to `rails notes --help` for more information.\n")
```
Comment (rafaelfranca) [inline railties/lib/rails/source_annotation_extractor.rb]: Here should be rake task.

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: railties/lib/rails/source_annotation_extractor.rb (hunk 1 above)
Comment (albertoalmagro) [reply railties/lib/rails/source_annotation_extractor.rb]: So true... 😮 Good catch @rafaelfranca 

Changing it.

**Comment by schneems** [comment]
Thanks!

**Pull request merged**